shims.d.ts

```diff
@@ -1,20 +0,0 @@
-// Auto-generated. Do not edit.
-declare namespace mlrunner {
-
-    /**
-     * Register a TypeScript function to run when an event is raised.
-     *
-     * This custom version of the MakeCode onEvent function is needed due to:
-     * https://github.com/microsoft/pxt-microbit/issues/5709
-     * 
-     *
-     * @param src The ID of the component to listen to.
-     * @param value The event value to listen to from that component.
-     * @param handler The function to call when the event is detected.
-     * @param flags The specified event flags are ignored and configured via pxt.json.
-     */
-    //% flags.defl=0 shim=mlrunner::customOnEvent
-    function customOnEvent(src: int32, value: int32, handler: () => void, flags?: int32): void;
-}
-
-// Auto-generated. Do not edit. Really.
```

Comment:
Have you tried to build the extension locally, it might complain if this file is not present, even if empty.
I don't know if loading the extension in the browser will have the same complain.

Reply:
It seems to be fine building it locally. It's not re-created and it doesn't complain about it being missing from the file list pxt.json. I would welcome a second opinion if you can try building it as well.

Reply:
Of course, just tried it and it builds on my dev environment as well 👍